### PR TITLE
Add wcloud events, update flags and error nicely when there is no config

### PR DESCRIPTION
**Pull request by tomwilkie**
Fixes https://github.com/weaveworks/deploy/issues/25

### CODE REVIEW

**Comment by jml** [comment]
LGTM.

File: cmd/wcloud/cli.go [hunk 1]
```diff
@@ -80,14 +82,17 @@ func deploy(c Client, args []string) {
 }
 
 func list(c Client, args []string) {
-	flags := flag.NewFlagSet("list", flag.ContinueOnError)
-	page := flags.Int("page", 0, "Zero based index of page to list.")
-	pagesize := flags.Int("page-size", 10, "Number of results per page")
+	var (
+		flags = flag.NewFlagSet("", flag.ContinueOnError)
+		since = flags.Duration("since", 7*24*time.Hour, "How far back to fetch results")
```
Comment (paulbellamy) [inline cmd/wcloud/cli.go]: I kind of expect since to be a timestamp, but not sure I have a better name.

File: cmd/wcloud/cli.go [hunk 2]
```diff
@@ -109,6 +114,26 @@ func list(c Client, args []string) {
...
+	)
+	if err := flags.Parse(args); err != nil {
+		usage()
+		return
+	}
+	through := time.Now()
+	from := through.Add(-*since)
```
Comment (paulbellamy) [inline cmd/wcloud/cli.go]: eugh, go.

**Comment by paulbellamy** [comment]
Looks fine.

File: cmd/wcloud/cli.go (hunk 1 above)
Comment (tomwilkie) [inline cmd/wcloud/cli.go]: I thought duration would be easier.  We can try out different things.

**Pull request merged**